python/benchmark/databricks/README.md

```diff
@@ -56,6 +58,6 @@ This directory contains shell scripts for running larger scale benchmarks on Dat
   databricks jobs cancel-run <runid> --profile $DB_PROFILE
   ```
 
-5. The created clusters are configured to terminate after 30 min, but can be manually terminated or deleted via the Databricks UI.
+1. The created clusters are configured to terminate after 30 min, but can be manually terminated or deleted via the Databricks UI.
 
-6. Monitor progress periodically in case of a possible hang, to avoid incurring cloud costs in such cases.
+2. Monitor progress periodically in case of a possible hang, to avoid incurring cloud costs in such cases.
```

Comment:
Do we keep item number 5, 6 or replace with 1, 2?

Reply:
Interesting.  Didn't intend to change the numbers, will revert.